accelerometer_chip_block.py

```diff
@@ -5,23 +5,18 @@
 import threading
 import time
 
+from . import adxl345
 from nio.common.block.base import Block
-from nio.common.signal.base import Signal
 from nio.metadata.properties.select import SelectProperty
-from nio.metadata.properties.list import ListProperty
-from nio.metadata.properties.string import StringProperty
 from nio.metadata.properties.int import IntProperty
 from nio.metadata.properties.string import StringProperty
-from nio.metadata.properties.bool import BoolProperty
 from nio.metadata.properties.timedelta import TimeDeltaProperty
-from nio.metadata.properties.holder import PropertyHolder
 from nio.common.discovery import Discoverable, DiscoverableType
-from nio.modules.scheduler import Job
-from nio.modules.threading import spawn
+
 
 def get_adxl345():
-    from .adxl345 import adxl345
```

Comment:
Given that their is only one Chip Type supported, this is ok, but the point of importing here was so that when new chips types are supported by the block, you don't need the python dependencies for all the other ones.